lib/passports.js

```diff
@@ -101,14 +101,26 @@ module.exports = {
                 passport.tokens = JSON.stringify(query.tokens)
               }
 
-              // Save any updates to the Passport before moving on
-              app.services.FootprintService.update('Passport', passport.id, passport) .then(passport => {
-                app.services.FootprintService.find('user', passport[0].user.id)
-                  .then(user => {
-                    next(null, user)
-                  })
-                  .catch(next)
-              }).catch(next)
+             // TODO: Must be tested for check if ok for both Sequelize and Waterline
+             // Save any updates to the Passport before moving on
+              app.services.FootprintService.find('Passport', passport.id)
```

Comment:
Ho ok ^^ :) I use it with sequelize only (I don't use waterline) with no problem but never use third party login like google auth/facebook/twitter... so yes maybe some problem may occur on this part as I don't know how unit test those... 
If you need help/feedbacks/advices, don't hesitate to reach me on gitter ;) 
I'll refuse this one and wait for the others then thanks !